### ISSUE TITLE
Command for retrieving non-state items from key-value store

### ISSUE THREAD

**Issue by mglaman** [drush-ops/drush#5669]
**Is your feature request related to a problem? Please describe.**
Drupal stores information in the `key_value` table. Drush already provides `state` commands to retrieve values from the `state` collection. However, Drupal uses its key-value store for more collections, such as `entity.definitions.installed` and `entity.storage_schema.sql`. It can be useful to retrieve these for debugging.

**Describe the solution you'd like**
Provide a command similar to `state` specifying the collection to use as an argument.

```shell
drush keyvalue:get COLLECTION NAME
```

**Describe alternatives you've considered**
Using `drush sqlq` but that does not unserialize the value and does not support the `json` format.

**Comment by Chi-teck** [comment]
Another alternative
```
drush ev 'echo \Drupal::keyValue("COLLECTION")->get("NAME")'
```

**Comment by mglaman** [comment]
Does `ev` support JSON formatting?

**Comment by Chi-teck** [comment]
No, it doesn't.

**Comment by weitzman** [comment]
FYI, formatters do work with `ev` if you return the object instead of echo. This classic gets DB data out as JSON - `drush ev --format=json "return \Drupal::service('database')->query('SELECT * FROM users LIMIT 5')->fetchAll()"`

I tried to do this with a complex object like `entity.definitions.installed` but formatting failed:

```
weitzman@drush-web:/var/www/html$ drush ev 'return \Drupal::keyValue("entity.definitions.installed")->get("user.entity_type")' --format=var_export

In FormatterManager.php line 231:

  The format var_export cannot be used with the data produced by this command, which was an instance of Drupal\Core\Entity\ContentEntityType.  Valid formats
  are: null,string
```

How should we transform a ContentEntityType into a json string?

**Comment by Chi-teck** [comment]
For objects that implement <code>EntityInterface</code> we could possible apply <code>toArray</code> method first.

**Comment by weitzman** [comment]
I made an output-formatters PR which handles the last 2 comments - https://github.com/consolidation/output-formatters/pull/109

**Comment by weitzman** [comment]
Above PR is merged and brought into Drush in #5685. A new Drush 12 release is imminent. Lets call this one done for now.
